src/jit/codegenxarch.cpp

```diff
@@ -6688,7 +6700,7 @@ void         CodeGen::genJumpKindsForTree(GenTreePtr    cmpTree,
 //------------------------------------------------------------------------
 // genJumpKindsForTreeLongHi: Generate the jump types for compare
 // operators of the high parts of a compare with long type operands
-// on x86
+// on x86 for uncontained compares
```

Comment:
It is confusing to use the word 'uncontained compare'  for the reason that isContained() always returns false on rel-op  (or compare) tree nodes.  What we here really intend to mean is for the case where rel-op result needs to be materialized into a register.

I would suggest to use "for the case where rel-op result needs to be materialized into a register"